modules/nw-ptp-configuring-linuxptp-services-as-boundary-clock.adoc

```diff
@@ -52,127 +54,128 @@ spec:
+    match:                                            <18>
+    - nodeLabel: "<node_label>"                       <19>
+      nodeName: "<node_name>"                         <20>
 ----
 <1> The name of the `PtpConfig` CR.
 <2> Specify an array of one or more `profile` objects.
 <3> Specify the name of a profile object which uniquely identifies a profile object.
 <4> This field should remain empty for boundary clock.
-<5> Specify system config options for the `ptp4l` service, for example `-2`. The options should not include the network interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because the network interface name and the service config file are automatically appended.
+<5> Specify system config options for the `ptp4l` service. The options should not include the network interface name `-i <interface>` and service config file `-f /etc/ptp4l.conf` because the network interface name and the service config file are automatically appended.
 <6> Specify the needed configuration to start `ptp4l` as boundary clock. For example, `ens1f0` synchronizes from a grandmaster clock and `ens1f3` synchronizes connected devices.
 <7> The interface that receives the synchronization clock.
-<8> The interface that synchronizes downstream connected devices.
-<9> Specify system config options for the `phc2sys` service, for example `-a -r`. If this field is empty the PTP Operator does not start the `phc2sys` service.
-<10> Scheduling policy for ptp4l and phc2sys processes. Default value is `SCHED_OTHER`. Use `SCHED_FIFO` on systems that support FIFO scheduling.
-<11> Integer value from 1-65 used to set FIFO priority for `ptp4l` and `phc2sys` processes. Required if `SCHED_FIFO` is set for `ptpSchedulingPolicy`.
-<12> Optional. If `ptpClockThreshold` stanza is not present, default values are used for `ptpClockThreshold` fields. Stanza shows default `ptpClockThreshold` values.
-<13> Specify an array of one or more `recommend` objects that define rules on how the `profile` should be applied to nodes.
-<14> Specify the `profile` object name defined in the `profile` section.
-<15> Specify the `priority` with an integer value between `0` and `99`. A larger number gets lower priority, so a priority of `99` is lower than a priority of `10`. If a node can be matched with multiple profiles according to rules defined in the `match` field, the profile with the higher priority is applied to that node.
-<16> Specify `match` rules with `nodeLabel` or `nodeName`.
-<17> Specify `nodeLabel` with the `key` of `node.Labels` from the node object by using the `oc get nodes --show-labels` command.
-<18> Specify `nodeName` with `node.Name` from the node object by using the `oc get nodes` command.
+<8> The interface that sends the synchronization clock.
+<9> For Intel Columbiaville 800 Series NICs, set `tx_timestamp_timeout` to `50`.
+<10> For Intel Columbiaville 800 Series NICs, set `boundary_clock_jbod` to `0`.
```

Comment:
In  your example above it is set to 0

Reply:
boundary_clock_jbod = 0 for BC is what we want here isn't it?